docs/persistenceClustering1.md

```diff
@@ -0,0 +1,62 @@
+# Persistence Clustering 1 Example
+
+<!--[![Dragon example video tutorial](https://topology-tool-kit.github.io/img/gallery/dragon.jpg)](https://youtu.be/YVk9vRKIEX8)-->
+
+<!--iframe width="100%" height="420"
+src="https://www.youtube.com/embed/YVk9vRKIEX8" frameborder="0"
+allowfullscreen></iframe-->
+
+![Persistence Clustering 1 Image](https://topology-tool-kit.github.io/img/gallery/persistenceClustering1.jpeg)
+
+## Pipeline description
+<!--This example first loads a triangle mesh from disk.-->
+First, this example loads a point cloud from disk (top left view in the above screenshot), then it computes a mesh on which a density field is obtained with a Gaussian Resampling on the points (top right view in the above screenshot). This density field will be considered as the input scalar data.
+<!--In a pre-processing, the mesh is smoothed and an elevation function is computed on top of it.-->
+<!--Then an elevation function is computed on it, and will be considered as the input scalar data for ou.-->
+
+Next, a [PersistenceDiagram](https://topology-tool-kit.github.io/doc/html/classttkPersistenceDiagram.html) is computed and thresholds are applied base on persistence to maintain only the features with a persistence above a certain value. The result is a simplified persistence diagram (bottom left view in the above screenshot).
+
+The simplified persistence diagram is then used as a constraint for the [TopologicalSimplification](https://topology-tool-kit.github.io/doc/html/classttkTopologicalSimplification.html) of the input scalar data, giving us a simplified data.
+
+Form there a [MorseSmaleComplex](https://topology-tool-kit.github.io/doc/html/classttkMorseSmaleComplex.html) is computed (bottom right view in the above screenshot) then smoothed. Finally, by using the identifier of the 2-dimension cell of the Morse Smale complex where one point lands, a cluster is given to it.
```

Comment:
I don't believe it is necessary to mention the smoothing of the Morse-Smale complex.

Reply:
Can you please also made it explicit which field encodes the output cluster Id?
(I believe this is `AscendingManifold`)